<?xml version="1.0" encoding="utf-8"?>
<robot name="target_object" xmlns:xacro="http://www.ros.org/wiki/xacro">


<xacro:include filename="$(find surgical_sim)/urdf/target_object/templates/gazebo.links.xacro"/>

  <xacro:macro name="create_base_object_links" params="parent_link xyz rpy">

    <!-- link 1: base_object_link -->
    <link name="base_object_link"/>
    <link name="target_object_link">
      <visual>
        <origin rpy="${-pi/2} 0 0" xyz="0 0 0"/>
        <geometry>
          <!-- <mesh filename="package://surgical_sim/model/meshes/dae/target_object/torso.dae"/>       -->
          <box size="0.050 0.050 0.050"/>
        </geometry>
      </visual>
      <!-- <inertial>
				<origin xyz="0 0 0" rpy="0 0 0"/>
				<mass value="1"/>
				<inertia ixx="1" ixy="0.0" ixz="0.0" iyy="1" iyz="0.0" izz="1"/>
			</inertial>  -->
    </link>

   <xacro:gazebo_links name="base_object_link"/>
   <xacro:gazebo_links name="target_object_link"/>
 
  </xacro:macro>
  

</robot>
